lib/ConnectionPool.cc

```diff
@@ -107,7 +108,7 @@ Future<Result, ClientConnectionWeakPtr> ConnectionPool::getConnectionAsync(
     LOG_INFO("Created connection for " << logicalAddress);
```

Comment:
Maybe we can print `key` instead of `logicalAddress` now. Otherwise users might not know which specific connection is created for `logicalAddress`.

```c++
LOG_INFO("Created connection for " << key);
```

Reply:
It could be confusing though to see a `broker-5-N` in the log and will be mistaken by a different host name

Reply:
Okay. It makes sense.